File: pruebasProyecto/passenger.cpp
#include "passenger.h"

using namespace std;

// Constructor de la clase Passenger que llama al constructor de Person
Passenger::Passenger(int _id, const string& _name, const string& _lastName,
    const string& _birthdate, const string& _gender,
    const string& _address, int _phoneNumber, const string& _email,
    const string& _nationality, int _baggageAmount, const string& _medicalInformation)
    : Person(_id, _name, _lastName, _birthdate, _gender, _address, _phoneNumber, _email),
   nationality(_nationality), baggageAmount(_baggageAmount), medicalInformation(_medicalInformation) {
}
Passenger::Passenger(){}
// Métodos get de Passenger
string Passenger::getNationality()  { return nationality; }
int Passenger::getBaggageAmount()  { return baggageAmount; }
string Passenger::getMedicalInformation()  { return medicalInformation; }

// Métodos set de Passenger
void Passenger::setNationality(const string& _nationality) { nationality = _nationality; }
void Passenger::setBaggageAmount(int _baggageAmount) { baggageAmount = _baggageAmount; }
void Passenger::setMedicalInformation(const string& _medicalInformation) { medicalInformation = _medicalInformation; }

File: pythonPr/passenger.py
from person import Person

class Passenger(Person):
    def __init__(self,  _ident = None,  _name = None,  _lastName = None,  _birthdate = None,  _gender = None,  _address = None,  _phoneNumber = None,  _email = None,  _nationality = None,  _baggageAmount = None,  _medicalInformation = None):
        super().__init__(_ident, _name, _lastName, _birthdate, _gender, _address, _phoneNumber, _email)
        self.nationality = _nationality
        self.baggageAmount = _baggageAmount
        self.medicalInformation = _medicalInformation

    # Métodos get de Passenger
    def getNationality(self):
        return self.nationality

    def getBaggageAmount(self):
        return self.baggageAmount

    def getMedicalInformation(self):
        return self.medicalInformation

    # Métodos set de Passenger
    def setNationality(self, _nationality):
        self.nationality = _nationality

    def setBaggageAmount(self, _baggageAmount):
        self.baggageAmount = _baggageAmount

    def setMedicalInformation(self, _medicalInformation):
        self.medicalInformation = _medicalInformation